web_services/index.ipynb

```diff
@@ -105,6 +105,163 @@
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_13.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "3. To turn on the Fixed range temporal navigation, select the **Fixed range temporal navigation** icon ![Temporal Navigation Fixed Range](../_static/web_services/mTemporalNavigationFixedRange.svg) on the **Temporal Controller Panel** and set the time range for the layer features to be visualized. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_14.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "4. To turn on the Animated temporal navigation, select the **Animated temporal navigation** icon ![Temporal Navigation Animated](../_static/web_services/mTemporalNavigationAnimated.svg) on the **Temporal Controller Panel**, and set animation range and the time step for the animation.  \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_15.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_16.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "5. Click on the play icon ![play](../_static/web_services/forward.svg) to view the animation. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_17.gif\" alt=\"QGIS - WMS-T layers\" width=\"500\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**How to export the  animation from QGIS Temporal Controller**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. The animation can be exported from the QGIS temporal controller as a series of images. To export the animation click on the **Export animation** icon ![save](../_static/web_services/mActionFileSave.svg) on the **Temporal Controller Panel**.\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_18.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "2. In the **Export Map animation** dialogue, specify: \n",
+    "    * The naming template to be used for labelling the images. \n",
+    "    * The output directory where the images will be saved.\n",
+    "    * The map extent to be covered by the images. \n",
+    "    * The width and height of the images. \n",
+    "    * The temporal range and time step for the animation.\n",
+    "    \n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_19.png\" alt=\"QGIS - WMS-T layers\" width=\"300\">\n",
+    "\n",
+    "3. The exported images will be saved in the specified ouput directory. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_20.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "4. Using your preferred software you can use the images to create an animation."
```

Comment:
This can be a comment on the list item 3 and not a list item itself.

Reply:
Fixed this and added an example animation:  https://github.com/digitalearthafrica/deafrica-docs/pull/41/commits/24091f6cd89718efbc7d15423f7852c8a691390d